nservicebus/sqlserver/deployment-options.md

```diff
@@ -0,0 +1,61 @@
+---
+title: SQL Server transport deployment options
+summary: Describes available SQL Server transport deployment options
+tags:
+- SQL Server
+- Transports
+redirects:
+- nservicebus/sqlserver/multiple-databases
+---
+
+In default configuration SQL Server transport assumes that all tables used for storing messages are located in a single catalog and within a single schema. The configuration can be changed to partition message storage between different schemas and catalogs. The decision can me made on per queue basis (including error and audit queues).
+
+The supported deployment options are:
+ * ***default***: all queues are stored in a single catalog and a single schema (Versions 1.x and higher)
+ * ***multi-schema***: queues are stored in a single catalog but in more than one schema (Versions 2.1 and higher)
+ * ***multi-catalog***: queues are stored in multiple catalogs but on a single SQL Server instance. This mode is indirectly supported by using *multi-instance* option, and requires using DTC. In this document both options are covered under *multi-instance* term.
+ * ***multi-instance***: queues are stored in multiple catalogs on more than one SQL Server instance (Versions 2.1 to 2.x).
+
+NOTE: To properly identify the chosen deployment option all queues that the endpoint interacts with need to be taken into consideration, including error and audit queues. If either of them are stored in a separate SQL Server instance then the deployment is a *multi-instance* one.
+
+The transport will route messages to destination based on the configuration. If no specific configuration has been provided for a particular destination, the transport assumes the destination has the same configuration as the sending endpoint (i.e. identical schema, catalog and instance name). If the destination has a different configuration and it hasn't been provided, then exception will be thrown immediately, because the transport wouldn't be able to connect to the destination queue.
```

Comment:
>  then exception will be thrown immediately, because the transport wouldn't be able to connect to the destination queue.

do you mean, an exception will be thrown when the endpoint starts?